dotnet/src/Skills/Skills.OpenAPI/Extensions/KernelOpenApiExtensions.cs

```diff
@@ -214,7 +214,7 @@ public static class KernelOpenApiExtensions
             try
             {
                 kernel.Log.LogTrace("Registering Rest function {0}.{1}", skillName, operation.Id);
-                var function = kernel.RegisterRestApiFunction(skillName, operation, authCallback, cancellationToken);
+                var function = kernel.RegisterRestApiFunction(skillName, operation, authCallback, null, cancellationToken);
```

Comment:
[nit] null parameter value doesn't give the reader much of hint of what is being null'd out. I would recommend using parameter names (e.g. `userAgent: null`).